packages/compass/src/app/components/home.tsx

```diff
@@ -173,60 +134,37 @@ function Home({
     [setIsWelcomeOpen, showSettings]
   );
 
-  const multiConnectionsEnabled = usePreference(
-    'enableMultipleConnectionSystem'
-  );
-
   return (
-    <FileInputBackendProvider createFileInputBackend={createFileInputBackend}>
-      <ConnectionImportExportProvider>
-        <CompassInstanceStorePlugin>
-          <FieldStorePlugin>
-            <div data-testid="home" className={verticalSplitStyles}>
-              {multiConnectionsEnabled && (
+    <CompassConnections
```

Comment:
Is this right now? The modal still doesn't seem to appear in the right place in the DOM.